languages-and-frameworks/crystal.html.md.erb

```diff
@@ -196,10 +196,10 @@ Fly also supports multi-region deployments.
 
 To deploy to multiple regions, first [Create a PostgreSQL Cluster](/docs/postgres/), then follow these steps:
 
-1. Configure your primary region by setting the env `PRIMARY_REGION` in your fly.toml
+1. Configure your primary region by setting the `primary_region` in your fly.toml. The `fly deploy` command uses `primary_region` to determine where to create new Machines, as well as to set the `PRIMARY_REGION` environment variable within the Machines.
+
   ```toml
-  [env]
-    PRIMARY_REGION = "ord"
+  primary_region = "ord"
```

Comment:
Another case of an old config that's only doing an environment variable, not an app primary_region setting.